db/seeds.rb

```diff
@@ -6,6 +6,14 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+User.create!(
+  email: "rovaira@gmail.com",
+  username: "rovaira",
+  password: "testtest",
+  password_confirmation: "testtest",
+  role: "admin"
+)
```

Comment:
It would be preferable to use the `find_or_create_by!` method here, as to not create duplicate entries. See the [Horizon article on seeding](https://horizon.launchacademy.com/lessons/seeding).